findmin.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import("fmt")
+
+func findmin(in []int) int{
+	smallest := in[0]
+    for i := 0; i < len(in); i ++ {
+        var x = in[i]
+
+        if x <= smallest {
+            smallest = x
```

Comment:
Hi. No need to assign new smallest value, if x  == smallest.

x < smallest
will do